course6/toyapp_ariel/CMakeLists.txt.user

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do not keep this file in a repo; this is not part of the code, but rather description of your local setup.
The file is completely unusable outside of the environment it's been generated